docs/virtual_machines/run_strategies.md

```diff
@@ -18,22 +18,26 @@ has been introduced. This is mutually exclusive with `Running` as they
 have somewhat overlapping conditions. There are currently four
 RunStrategies defined:
 
--   Always: A VirtualMachineInstance will always be present. If the
-    VirtualMachineInstance crashed, a new one will be spawned. This is
-    the same behavior as `spec.running: true`.
-
--   RerunOnFailure: A VirtualMachineInstance will be respawned if the
-    previous instance failed in an error state. It will not be
-    re-created if the guest stopped successfully (e.g. shut down from
-    inside guest).
-
--   Manual: The presence of a VirtualMachineInstance or lack thereof is
-    controlled exclusively by the start/stop/restart VirtualMachine
-    subresource endpoints.
-
--   Halted: No VirtualMachineInstance will be present. If a guest is
-    already running, it will be stopped. This is the same behavior as
-    `spec.running: false`.
+-   Always: The system is asked to always have the VM running.
```

Comment:
I would write this as "The system is tasked with keeping the VM in the running state."

Completely optional change, it may only be a matter of style.